ndk/src/aaudio.rs

```diff
@@ -92,7 +92,7 @@ pub enum AAudioFormat {
     /// Values outside that range may be clipped.
     ///
     /// See also 'floatData' at
-    /// https://developer.android.com/reference/android/media/AudioTrack#write(float[],%20int,%20int,%20int)
+    /// <https://developer.android.com/reference/android/media/AudioTrack#write(float[],%20int,%20int,%20int)>
```

Comment:
Note that this link is broken; rustdoc rightfully converts `[]` to `%5B%5D`, but android.com does not resolve that to the right anchor.